### PR TITLE
[next] tree: promote changes from next-devel at e87dc88

### PR DESCRIPTION
**Pull request by coreosbot-releng**
Triggered by @dustymabe in https://github.com/coreos/fedora-coreos-streams/issues/377.

### CODE REVIEW

**Review by jlebon — APPROVED** [review]
```diff
$ git diff next-devel
diff --git a/manifest.yaml b/manifest.yaml
index a768ebd..aae7965 100644
--- a/manifest.yaml
+++ b/manifest.yaml
@@ -1,4 +1,4 @@
-ref: fedora/${basearch}/coreos/next-devel
+ref: fedora/${basearch}/coreos/next
 include: manifests/fedora-coreos.yaml

 releasever: "35"
@@ -6,34 +6,10 @@ releasever: "35"
 rojig:
   license: MIT
   name: fedora-coreos
-  summary: Fedora CoreOS next-devel
-
-repos:
-  # These repos are there to make it easier to add new packages to the OS and to
-  # use `cosa fetch --update-lockfile`; but note that all package versions are
-  # still pinned. These repos are also used by the remove-graduated-overrides
-  # GitHub Action.
-  - fedora-next
-  - fedora-next-updates
-
-# All Fedora CoreOS streams share the same pool for locked files.
-# This will be in fedora-coreos.yaml in the future so it can be more easily be
-# shared between all the streams
-lockfile-repos:
-  - fedora-coreos-pool
+  summary: Fedora CoreOS next

 add-commit-metadata:
-  fedora-coreos.stream: next-devel
-
-postprocess:
-  # Disable Zincati and fedora-coreos-pinger on non-production streams
-  # https://github.com/coreos/fedora-coreos-tracker/issues/163
-  - |
-    #!/usr/bin/env bash
-    set -xeuo pipefail
-    mkdir -p /etc/fedora-coreos-pinger/config.d /etc/zincati/config.d
-    echo -e '# https://github.com/coreos/fedora-coreos-tracker/issues/163\nreporting.enabled = false' > /etc/fedora-coreos-pinger/config.d/90-disable-on-non-production-stream.toml
-    echo -e '# https://github.com/coreos/fedora-coreos-tracker/issues/163\nupdates.enabled = false' > /etc/zincati/config.d/90-disable-on-non-production-stream.toml
+  fedora-coreos.stream: next

 packages:
   # resolved was broken out to its own package in rawhide/f35
```

Ahh right OK, we need to drop the `lockfile-repos` bit from next-devel. (Edit: https://github.com/coreos/fedora-coreos-config/pull/1236)
But this PR itself LGTM!

File: manifest-lock.x86_64.json [hunk 1]
```diff
@@ -1,1205 +1,1211 @@
...
-      "evra": "18-1.fc34.x86_64"
+      "evra": "18-3.fc35.x86_64"
     },
     "cloud-utils-growpart": {
-      "evra": "0.31-8.fc34.noarch"
-    },
-    "compat-readline5": {
-      "evra": "5.2-39.fc34.x86_64"
+      "evra": "0.31-9.fc35.noarch"
     },
```
Comment (dustymabe) [inline manifest-lock.x86_64.json:110]: `compat-readline5` was a dep of `lvm2` but no longer as it now uses editline (from `libedit`, which is already part of FCOS as a dep of other software). https://src.fedoraproject.org/rpms/lvm2/c/24098c7fd8e29251b5884ff5db0db61abcf2ce22?branch=rawhide

Comment (dustymabe) [reply manifest-lock.x86_64.json:110]: also: https://github.com/coreos/fedora-coreos-config/pull/1206/files#r705430374

File: manifest-lock.x86_64.json [hunk 2]
```diff
@@ -1,1205 +1,1211 @@
...
-      "evra": "055-3.fc34.x86_64"
+      "evra": "055-3.fc35.1.x86_64"
     },
     "dracut-network": {
-      "evra": "055-3.fc34.x86_64"
+      "evra": "055-3.fc35.1.x86_64"
+    },
+    "dracut-squash": {
+      "evra": "055-3.fc35.1.x86_64"
     },
```
Comment (dustymabe) [inline manifest-lock.x86_64.json:239]: dracut-squash added back in kexec-tools https://github.com/coreos/fedora-coreos-config/pull/1206/files#r705426948

File: manifest-lock.x86_64.json [hunk 3]
```diff
@@ -1,1205 +1,1211 @@
...
-      "evra": "1.5.12-1.fc34.x86_64"
+      "evra": "1.6.3-1.fc35.x86_64"
     },
     "gawk": {
-      "evra": "5.1.0-3.fc34.x86_64"
+      "evra": "5.1.0-4.fc35.x86_64"
+    },
+    "gdbm-libs": {
+      "evra": "1:1.20-2.fc35.x86_64"
     },
```
Comment (dustymabe) [inline manifest-lock.x86_64.json:344]: https://github.com/coreos/fedora-coreos-config/pull/1206/files#r705433251

File: manifest-lock.x86_64.json [hunk 4]
```diff
@@ -1,1205 +1,1211 @@
...
     "iproute-tc": {
-      "evra": "5.10.0-2.fc34.x86_64"
+      "evra": "5.13.0-2.fc35.x86_64"
     },
-    "iptables": {
-      "evra": "1.8.7-3.fc34.x86_64"
+    "iptables-legacy-libs": {
+      "evra": "1.8.7-13.fc35.x86_64"
     },
```
Comment (dustymabe) [inline manifest-lock.x86_64.json:422]: The `iptables` legacy binaries, libraries, and headers were moved into sub-packages in https://src.fedoraproject.org/rpms/iptables/c/f04bde23a01002f45c03863516e994c7332a37b3?branch=rawhide

I wonder if we should be installing `iptables-legacy` rpm now or if we should YOLO and let `iptables-nft` take over. At least https://github.com/coreos/fedora-coreos-tracker/issues/676 would be taken care of.

---

`iptables-legacy-libs` getting pulled in by systemd:

```
[core@cosa-devsh ~]$ rpm -e iptables-legacy-libs
error: Failed dependencies:
        libip4tc.so.2()(64bit) is needed by (installed) systemd-249.4-1.fc35.x86_64
```

File: manifest-lock.x86_64.json [hunk 5]
```diff
@@ -1,1205 +1,1211 @@
...
-      "evra": "1.31-3.fc34.x86_64"
+      "evra": "1.31-4.fc35.x86_64"
     },
     "libtevent": {
-      "evra": "0.11.0-0.fc34.x86_64"
-    },
-    "libtextstyle": {
-      "evra": "0.21-4.fc34.x86_64"
+      "evra": "0.11.0-1.fc35.x86_64"
     },
```
Comment (dustymabe) [inline manifest-lock.x86_64.json:770]: https://github.com/coreos/fedora-coreos-config/pull/1206/files#r705434440

File: manifest-lock.x86_64.json [hunk 6]
```diff
@@ -1,1205 +1,1211 @@
...
-      "evra": "1.7.4.1-2.fc34.x86_64"
+      "evra": "1.7.4.1-3.fc35.x86_64"
     },
     "sqlite-libs": {
-      "evra": "3.34.1-2.fc34.x86_64"
+      "evra": "3.36.0-3.fc35.x86_64"
+    },
+    "squashfs-tools": {
+      "evra": "4.5-2.fc35.x86_64"
     },
```
Comment (dustymabe) [inline manifest-lock.x86_64.json:1085]: https://github.com/coreos/fedora-coreos-config/pull/1206/files#r705435039

File: manifest-lock.x86_64.json [hunk 7]
```diff
@@ -1,1205 +1,1211 @@
...
     "userspace-rcu": {
-      "evra": "0.12.1-3.fc34.x86_64"
+      "evra": "0.13.0-3.fc35.x86_64"
     },
     "util-linux": {
-      "evra": "2.36.2-1.fc34.x86_64"
+      "evra": "2.37.2-1.fc35.x86_64"
+    },
+    "util-linux-core": {
+      "evra": "2.37.2-1.fc35.x86_64"
```
Comment (dustymabe) [inline manifest-lock.x86_64.json:1162]: https://github.com/coreos/fedora-coreos-config/pull/1206/files#r705435705

File: manifest-lock.x86_64.json (hunk 4 above)
Comment (jlebon) [reply manifest-lock.x86_64.json:422]: I think based on https://github.com/coreos/fedora-coreos-tracker/issues/676#issuecomment-742115786, it sounds like switching over upgrading nodes isn't safe to do.

**Pull request merged**